_infra/helm/locust/locustfiles/requests.json

```diff
@@ -0,0 +1,19 @@
+{
+    "requests": [
+        {
+            "method": "GET",
+            "url": "/surveys/todo",
+            "expected_response_text": "Click on the survey name to complete your questionnaire"
+        },
+        {
+            "method": "GET",
+            "url": "/surveys/history",
+            "expected_response_text": "aria-selected=\"true\" id=\"SURVEY_HISTORY_TAB\" role=\"tab\">History</a>"
```

Comment:
I can see why we've done this (given there's very little "History" specific text elements on the page to assert on).  Does the 'aria-selected' value get set by the design system component? If so, this is subject to change if they change the DS implementation.

Reply:
Done.